.azure/templates/build-config-user.yml

```diff
@@ -6,10 +6,11 @@ parameters:
   arch: ''
   tls: ''
   extraBuildArgs: ''
+  extraName: ''
 
 jobs:
-- job: build_${{ parameters.platform }}_${{ parameters.arch }}_${{ parameters.tls }}
-  displayName: Build ${{ parameters.platform }} ${{ parameters.arch }} ${{ parameters.tls }}
+- job: build_${{ parameters.platform }}_${{ parameters.arch }}_${{ parameters.tls }}_${{ parameters.extraName }}
+  displayName: Build ${{ parameters.platform }} ${{ parameters.arch }} ${{ parameters.tls }} ${{ parameters.extraName }}
```

Comment:
Sorry to keep asking you to change things, but architecturally, I think considering UWP a separate platform might be better; especially when considering directory structure. What do you think? I'm not sure how much of a pain it would be to change it that way though.

Reply:
If we're just considering it separate on Azure then its not too bad. In the src, its platform would still be Windows. vcpkg considers uwp a separate architecture inside of windows, which is why I went that way originally.

Reply:
I think integrating it into our PowerShell and AZP yml files as just a different platform might be best.

Reply:
I don't think you should need the `extraName` parameter any more, right? Also, I'd like it if we could infer that `extraBuildArgs` based on the platform now.